keps/sig-node/4369-allow-special-characters-environment-variable/README.md

```diff
@@ -226,7 +226,7 @@ Yes.
 
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-When a feature gate is closed, already running workloads are not affected in any way, but update fields for workload will cause the workload to fail.
+When a feature gate is disabled, already running workloads are not affected in any way, but update fields for workload will cause the workload to fail.
```

Comment:
What fields would change in an update for this?

Reply:
Any update to the workload's fields results in failure because pods need to be recreate.

Reply:
Maybe I missed this in the KEP but I don't understand how that is related.

Reply:
If we update any field of workloads that use an environment variable with relaxed validation when disabling feature gate, the workload might fail to recreate pods or ReplicaSets because it cannot pass the validation logic of the Apiserver.

Reply:
I see. So you are saying that if you have expanded environment variable support and then you downgrade it could be possible to start failing validation.

Your phrase above seems to be more general than what you expanded here.

Reply:
This is, unfortunately, true of any feature on pod - if you enable, use in Deployment, disable, then the ".. or already in use" logic doesn't matter much.